modules/nw-egress-router-about.adoc

```diff
@@ -44,14 +43,6 @@ In _redirect mode_, an egress router pod configures `iptables` rules to redirect
 $ curl <router_service_IP> <port>
 ----
 
-ifdef::openshift-sdn[]
-In _HTTP proxy mode_, an egress router pod runs as an HTTP proxy on port `8080`. This mode only works for clients that are connecting to HTTP-based or HTTPS-based services, but usually requires fewer changes to the client pods to get them to work. Many programs can be told to use an HTTP proxy by setting an environment variable.
-
-In _DNS proxy mode_, an egress router pod runs as a DNS proxy for TCP-based services from its own IP address to one or more destination IP addresses. To make use of the reserved, source IP address, client pods must be modified to connect to the egress router pod rather than connecting directly to the destination IP address. This modification ensures that external destinations treat traffic as though it were coming from a known source.
-
-Redirect mode works for all services except for HTTP and HTTPS. For HTTP and HTTPS services, use HTTP proxy mode. For TCP-based services with IP addresses or domain names, use DNS proxy mode.
-endif::openshift-sdn[]
-
 ifdef::ovn[]
```

Comment:
I left them because HCM formerly ROSA uses some of our docs. Once I confirm with them, I've got a follow up PR that removes metadata from the docs and will handle those in that PR SME and QE aren't necessary in that case.